applications/FluidDynamicsApplication/tests/TwoFluidMassConservationTest/3Dtest.json

```
@@ -0,0 +1,81 @@
+    "solver_settings"             		: {
+        "model_part_name" 	: "FluidModelPart",
+        "domain_size"     	: 3,
+        "solver_type"   	: "TwoFluids",
+        "model_import_settings"       	: {
+            "input_type"     : "mdpa",
+            "input_filename" : "TwoFluidMassConservationProcTest/3Dtest"
+        },
+        "material_import_settings"       	: {
+            "materials_filename" : "TwoFluidMassConservationProcTest/TwoFluidMassConservation3DTestMaterials"
+        },
+    	"maximum_iterations"			: 5,
+    	"echo_level"					: 0,
+    	"time_order"					: 2,
+    	"compute_reactions"				: false,
+    	"reform_dofs_at_each_step"		: false,
+    	"relative_velocity_tolerance"	: 0.001,
+    	"absolute_velocity_tolerance"	: 0.00001,
+    	"relative_pressure_tolerance"	: 0.001,
+    	"absolute_pressure_tolerance"	: 0.00001,
+        "volume_model_part_name"     	: "Parts_Fluid",
+        "skin_parts"                 	: ["AutomaticInlet3D_Inlet","Outlet3D_Outlet"],
+        "no_skin_parts"              	: [],
+        "move_mesh_flag"				: false,
+		"formulation": {
+			"dynamic_tau": 1.0
+		},
+        "time_stepping"  : {
+            "automatic_time_step" 	: false,
+            "time_step"           	: 0.025
+        },
+    	"linear_solver_settings": {
+        	"solver_type": "amgcl"
+    	}
+    },
+	"processes"	:	{
+    	"initial_conditions_process_list"  : [],
+    	"boundary_conditions_process_list" : [{
+            "python_module" : "apply_inlet_process",
+            "kratos_module" : "KratosMultiphysics.FluidDynamicsApplication",
+            "Parameters"    : {
+                "model_part_name" : "FluidModelPart.AutomaticInlet3D_Inlet",
+                "variable_name"   : "VELOCITY",
+                "modulus"         : 1.0,
+                "direction"       : [1.0,0.0,0.0],
+                "interval"        : [0.0,"End"]
+            }
+        },{
+            "python_module" : "apply_outlet_process",
+            "kratos_module" : "KratosMultiphysics.FluidDynamicsApplication",
+            "Parameters"    : {
+                "model_part_name"    : "FluidModelPart.Outlet3D_Outlet",
+                "variable_name"      : "PRESSURE",
+                "constrained"        : true,
+                "value"              : 0.0,
+                "hydrostatic_outlet" : false,
+                "h_top"              : 0.0
+            }
+        }],
+    	"auxiliar_process_list"            : [{
+        	"python_module" : "apply_mass_conservation_check_process",
+        	"kratos_module" : "KratosMultiphysics.FluidDynamicsApplication",
+        	"process_name"  : "ApplyMassConservationCheckProcess",
+        	"Parameters"    : {
+    			"model_part_name"						: "FluidModelPart",
+				"perform_corrections"					: true,
+    			"correction_frequency_in_time_steps"	: 2,
+    			"write_to_log_file"						: true,
+    			"log_file_name"							: "TwoFluidMassConservationProcTest/mass_cons_3D.log"
+        	}
+    	}]
```

```suggestion
    	"auxiliar_process_list"            : []
```
This shouldn't be no longer required right?